src/Controller/EnterController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+/**
+ * Контролер для обробки входу користувачів.
+ */
+class EnterController extends AbstractController
+{
+    /**
+     * Відображає сторінку входу та обробляє введені дані.
+     *
+     * @param Request $request HTTP-запит із введеними користувачем даними.
+     * @return Response Відповідь із відрендереним шаблоном або перенаправленням.
```

Comment:
Анотації завзвичай автозгенеровані і парсяться IDE або деколи навіть фреймворками. Ці стрічки коду не повинні містити додатковий текст.
```suggestion
     * @param Request $request
     * @return Response